config.py

```diff
@@ -0,0 +1,19 @@
+import os
+from importlib import import_module
+
+class Config(object):
+    SYS_IM_TYPE = os.environ.get("SYS_IM_TYPE")
```

Comment:
I think you can use gflags to handle config?
https://github.com/google/python-gflags

Reply:
Maybe next time. I think It is just  a simple config. Let user check by themselves.  If wrong, then server can't start.

Reply:
haha yeap~